keras_retinanet/bin/train.py

```diff
@@ -109,14 +109,31 @@ def create_callbacks(model, training_model, prediction_model, validation_generat
         checkpoint = RedirectModel(checkpoint, prediction_model)
         callbacks.append(checkpoint)
 
+    if args.tensorboard_dir:
+        tensorb = keras.callbacks.TensorBoard(
```

Comment:
I prefer the name `tensorboard` instead.

Reply:
Or perhaps `tb_callback` to more closely reflect what the Object is?

Reply:
I'd prefer `tensorboard_callback` then. `tb` is a bit cryptic to me.